### PR TITLE
Feature/lesson03 exercise uang

### PR DESCRIPTION
**Pull request by kenichigouang**

### CODE REVIEW

File: src/test/groovy/com/synacy/lesson03/exercise/EnrollmentServiceSpec.groovy
Comment (edwardGolez) [inline src/test/groovy/com/synacy/lesson03/exercise/EnrollmentServiceSpec.groovy]: Testing with collections is better when you have at least two. For this instance, we would like to gain confidence that **_all_** `CourseClass`es within `EnrollmentDto` are enrolled by the student. In that case, we would not be sure if all classes in the collection were enrolled if we only have 1 in our test case scenario.

**Review by edwardGolez — APPROVED** [review]

**Comment by kenichigouang** [comment]
@edwardGolez I made the changes Paul. Should I wait for your review before merging?

**Pull request closed without merging**
